internal/clients/provider_config.go

```diff
@@ -57,7 +57,7 @@ const (
 
 // userAgentV2 constructs the Crossplane user agent for AWS v2 clients
 var userAgentV2 = config.WithAPIOptions([]func(*middleware.Stack) error{
-	awsmiddleware.AddUserAgentKeyValue("upbound-provider-aws", version.Version),
+	awsmiddleware.AddUserAgentKeyValue("crossplane-provider-aws-upbound", version.Version),
```

Comment:
@nabuskey do you have an example of what the community `crossplane-contrib/provider-aws` user agent looks like?

Reply:
It's set here: 
https://github.com/crossplane-contrib/provider-aws/blob/fde8c4d07eeb6d78cbcb55729fefa733622a409b/pkg/clients/aws.go#L90

And it looks like this in CloudTrail.
```
aws-sdk-go-v2/1.16.7 os/linux lang/go/1.18.6 md/GOOS/linux md/GOARCH/amd64 api/sts/1.12.0 crossplane-provider-aws/v0.33.0
```